artview/components/plot.py

```diff
@@ -61,13 +61,14 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None, \
         parent - PyQt instance
             Parent instance to associate to Display window.
             If None, then Qt owns, otherwise associated with parent PyQt instance.
-        
+
         Notes::
         -----
         This class records the selected button and passes the 
         change value back to variable.
         '''
         super(Display, self).__init__(name=name, parent=parent)
+        self.setFocusPolicy(QtCore.Qt.ClickFocus)
```

Comment:
This is probably system dependent, but at one point I could not use zoom because the mouse wheel was changing focus. So this what the solution I found. I believe the standard configuration is [Qt.WheelFocus](http://pyqt.sourceforge.net/Docs/PyQt4/qt.html#FocusPolicy-enum)